cli/ballerina-cli/src/main/resources/cli-help/ballerina-persist.help

```diff
@@ -5,20 +5,18 @@ SYNOPSIS
        bal persist <command> [<-h> | <--help>]
 
 DESCRIPTION
-       Manage data persistence in a Ballerina package.
+       Manage the data persistence in a Ballerina package.
 
-       Initializing the package for persistence is the first step towards achieving
+       Initializing the package for persistence is the first step toward achieving
        data persistence in a Ballerina package. This step creates the required
-       directories and configuration files.
-       For example:
+       directories and configuration files. For example:
```

Comment:
```suggestion
       directories and updates the configuration files to integrate the generation of client API into the package build process. For example:
```

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/commit/ef73dd6fdbd23ed185882b2a23767ffaa7c67725